src/components/item/ItemPreviewCard.vue

```diff
@@ -11,27 +11,53 @@
     :blank-image-height="280"
     :variant="variant"
     :lazy="lazy"
+    :sub-title="subTitle"
   >
     <template #buttons>
-      <RecommendationButtons
-        v-if="enableAcceptRecommendation || enableRejectRecommendation"
-        :identifier="identifier"
-        :enable-accept-button="enableAcceptRecommendation"
-        :enable-reject-button="enableRejectRecommendation"
-      />
-      <UserButtons
-        v-else
-        :identifier="identifier"
-        :show-pins="showPins"
-        @like="$emit('like', identifier)"
-        @unlike="$emit('unlike', identifier)"
-      />
+      <div
+        v-if="variant === 'list'"
+        class="data-and-buttons-wrapper d-flex flex-wrap"
+      >
+        <RightsStatement
+          v-if="rights"
+          :rights-statement-url="rights"
+          variant="simple"
+        />
+        <span
```

Comment:
Is the closing </span> tag for this missing?

Reply:
The closing tag is on line 30, a bit confusing because of the self-closing span tag on the same line, I'll put it on a new line.

Reply:
Right you are. Thanks!